.crews.php

```diff
@@ -295,7 +296,12 @@ function main () {
                 }
                 $statement->bindValue(':date', $date->format('Y-m-d'));
                 $statement->execute();
-                $y = $statement->fetchAll(PDO::FETCH_ASSOC)[0];
+                $y = $statement->fetchAll(PDO::FETCH_ASSOC);
+
+                if (sizeof($y) == 0) {
```

Comment:
It's suggested to use `count` instead of `sizeof` (the latter is an alias of the former) to avoid confusion with other languages where `sizeof` means something different.

Reply:
Sizeof was used elsewhere in this code for similar uses and likely exists elsewhere. Probably a good idea to change it universally if it is going to be changed